crates/turborepo-lib/src/package_graph/mod.rs

```diff
@@ -26,6 +28,7 @@ pub struct PackageGraph {
 pub struct Entry {
     package_json: PackageJson,
     package_json_path: AnchoredSystemPathBuf,
+    turbo_json: TurboJson,
```

Comment:
I feel like this colocation is unnecessary and restricts us a bit as it means that we need to eagerly parse all `turbo.json` in order to construct a package graph and for some operations this isn't necessary.